IssueTrackerApplication/Authorization/OAuthAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace IssueTracker.Application.Authorization
+{
+    public class OAuthAttribute : TypeFilterAttribute
+    {
+        public string[] Scopes;
```

Comment:
```suggestion
        public string[] Scopes { get; private set; }
```

please add an empty line between methods, properties, etc.

Reply:
Added this change to the latest commit.